projects/quests.md

```diff
@@ -28,6 +30,105 @@ A quest that is intended to be completed only once would define in the payment c
 
 The evidence required for completing a quest will depend on the work expected. For a quest on writing a blog post, the evidence required will likely include a link to the blog post. It may also specify the minimum length of the blog post and that it hasn't been copied from somewhere else. For a quest on adding a feature to a web UI, the evidence required will likely include a link to the code written, some screenshots of the deployed feature and some reference to the code standard expected.
 
+## Pages
+
+### List view
+
+![](https://i.imgur.com/hRtXi66.png)
+
+This page consists of a small dashboard and a list of summarized Quests.
+
+The dasboard shows the following information:
+
+- Total of the active Quests bounties in USD
+- Number of active Quests
+- A **Create quest** button
+
+This list can be filtered by Quest title, description, expiration date and state.
+
+The expired Quests can be accessed by selecting **Expired** in the state filter.
+
+### Detail view
+
+This page is the detailed version of the quest showing more information :
+
+- The claim deposit (amount locked during the claiming process)
+- The full description (not summarized)
+- The list of the **Players**' claims
+- The action buttons (needs to be connected)
+  - Fund (where Patrons can fund the quest)
+  - Schedule claim (where Players can submit their evidence of task completion)
+
+![](https://i.imgur.com/HlJA7fs.png)
+
+## Create a quest
+
+**Creators** can create a Quest only when connected to a wallet. The **Create quest** buttons will then appear in the header/dashboard/footer.
+
+The creator must submit the following in 2 steps :
+
+1. First:
+   - Title
+   - [Description](quests.md#quest-descriptions) (the field supports markdown)
+2. Second:
+   - Funding token (any token address can be put in the field)
+   - Initial bounty
+   - Expire time (after that, Players will not be able to schedule new claims)
+   - Fallback address (the address where the funds can be restored once the quest expires)
+3. The Creator needs to deposit a collateral that will be locked until the Quest expiration date. He will need to Reclaim the deposit to recover it
+
+## Fund a quest
+
+**Patrons** can fund a Quest only when connected to a wallet. The Fund button is in the detail view of a Quest.
+
+The Patron should only submit the amount of funding for the Quest reward token.
```

Comment:
Its not clear, what it means?

Reply:
![image](https://user-images.githubusercontent.com/13890618/171967790-9aaca9d3-e302-42e6-aa6d-619c52db39b6.png)